lib/sht4x.ex

```diff
@@ -43,24 +76,48 @@ defmodule SHT4X do
   ## Callbacks
 
   @impl GenServer
+  @spec init(nil | maybe_improper_list | map) ::
+          {:ok,
+           %{
+             last_raw_measurement: nil,
+             current_measurement: nil,
+             options: keyword(),
+             serial_number: pos_integer(),
+             transport: SHT4X.Transport.t()
+           }}
+          | {:stop, :normal}
```

Comment:
```suggestion
```
Ugh. That spec. I assume autogenerated? I usually delete the `init/1` ones and rely on the `@impl`.